nautobot/docs/additional-features/jobs.md

```diff
@@ -509,6 +509,27 @@ When a new Job record is created for a newly discovered Job class, it defaults t
 
 An administrator or user with `extras.change_job` permission can edit the Job to change it to `enabled = True`, permitting running of the Job, when they have completed any appropriate review of the new Job to ensure that it meets their standards. Similarly, an obsolete or no-longer-used Job can be prevented from inadvertent execution by changing it back to `enabled = False`.
 
+ By default when a Job is installed into Nautobot it is installed in a disabled state. In order to enable a Job:
+
+ * Navigate to Jobs > Jobs menu
+ * Select a job that has been installed
+ * Select **Edit** button
+ * In the second section titled _Job_, select the **Enabled** checkbox
+ * Select **Update** button at the bottom
+
+### Enabling Job Hooks
```

Comment:
Do we need to broaden this heading to include Job Buttons?

Reply:
Also, should this be a level-4 heading (`####`) under the `Enabling Jobs for Running` parent heading?

Reply:
I probably should move it out. That was a copy and paste when I should have been sleeping.